packages/fast-tooling/src/web-components/css-layout/css-layout.template.ts

```diff
@@ -720,17 +729,18 @@ export const cssLayoutTemplate = (context: ElementDefinitionContext) => html<CSS
                     )}
                 </div>
             </div>
-            <div class="control-region">
+            <div class="controlRegion">
                 <label for="fast-tooling-css-align-items">Align Items</label>
-                <div class="control-radio-region">
+                <div class="radioRegion">
                     ${repeat(
                         x => x.alignItemsOptions,
                         html<string, CSSLayout>`
                             <div
-                                class="${(x, c) => c.parent.alignItemsName} ${x => x} ${(
-                            x,
-                            c
-                        ) => (x === c.parent.alignItemsValue ? "active" : "")}"
+                                class="radioRegion-div ${(x, c) =>
+                                    c.parent.alignItemsName} ${x => x} ${(x, c) =>
+                            x === c.parent.alignItemsValue
+                                ? "radioRegion-div__active"
```

Comment:
In general I think using elements like `input` and `textArea` is fine, because those are both the element name and the purpose of the item assigned with the class with those names, but `div` is inherently a blank slate, it could be doing anything.

Because this is in a `repeat` we could treat it like an item in a list, my suggestion would be:

```suggestion
                                ? "radioRegion-contentItem__active"
```

Reply:
Done